assets/src/scss/layout/_navbar.scss

```diff
@@ -196,22 +213,127 @@ $navbar-default-height: 60px;
     @include large-and-up {
       display: flex;
       flex-grow: 1;
-      justify-content: end;
+      align-items: center;
+      justify-content: flex-end;
       padding-inline-end: 0;
       padding-top: 0;
       text-align: end;
     }
   }
 }
 
+.nav-submenu {
+  _-- {
+    width: 193px;
+  }
+  position: fixed;
+  display: none;
+  padding-top: $sp-1;
+
+  .nav-submenu-wrapper {
+    border-radius: $sp-1;
+    width: 100%;
+    box-shadow: 0 1px $sp-x rgba(0, 0, 0, 0.12);
+  }
+
+  ul {
+    width: 100%;
+    // The `height` value represents the value of 6 items's height
+    // and this selectors's padding
+    height: auto;
+    max-height: 320px;
+    display: block;
+    flex-direction: column;
+    padding: $sp-2 0;
+    background: $white;
+    border-radius: $sp-1;
+    overflow-y: auto;
+
+    // Only applied to Firefox
+    scrollbar-width: thin;
+    scrollbar-color: transparent;
+
+    &::-webkit-scrollbar {
+      width: $sp-1;
```

Comment:
I don't think we should use these variables for `width` (nor for `border-radius` below)